src/virtualenv/seed/wheels/periodic_update.py

```diff
@@ -186,7 +186,7 @@ def trigger_update(distribution, for_py_version, wheel, search_dirs, app_data, p
     pipe = None if debug else subprocess.PIPE
     kwargs = {"stdout": pipe, "stderr": pipe}
     if not debug and sys.platform == "win32":
-        kwargs["creationflags"] = DETACHED_PROCESS
+        kwargs["creationflags"] = DETACHED_PROCESS | CREATE_NO_WINDOW
```

Comment:
This won't work - the `CREATE_NO_WINDOW` flag needs to be passed by itself, it's ignored if passed along with `DETACHED_PROCESS`.